docs/spec.md

```diff
@@ -60,6 +64,7 @@ Each JSON message must be an object containing a field called `op` which identif
   - `parameters`: Allow clients to get & set parameters
   - `parametersSubscribe`: Allow clients to subscribe to parameter changes
   - `time`: The server may publish binary [time](#time) messages
+  - `services`: Allow clients to call services
```

Comment:
Alternatively, if the server never advertises a service there are no services for the client to call :)

Reply:
```suggestion
  - `services`: Server can advertise services which clients may call
```

Reply:
This capability seems redundant. If a server can't advertise services, then it won't advertise any services. If it does advertise a service, why should a client make an additional check to ensure the `services` capability was set? And what should it do if a service is advertised, but the capability is not?

Reply:
It may help a client to know whether the server will potentially announce services or not. But I do not have a strong opinion on this. @jtbandes what do you think?